recipe/meta.yaml

```diff
@@ -11,11 +11,11 @@ source:
   url: https://github.com/jgm/pandoc/releases/download/{{ version }}/pandoc-{{ version }}-macOS.pkg  # [osx]
   sha256: a55c91f0bbb59246ac0ec841d68ac00b70e7e5a7af3f15c4fb25338fcfe1f0a7  # [osx]
 
-  url: https://github.com/jgm/pandoc/releases/download/{{ version }}/pandoc-{{ version }}-windows-x86_64.msi  # [win]
-  sha256: d11f3e08156c0544417af6a44cb44af8e5c17c0475663bebc046d395de4916dd  # [win]
+  url: https://github.com/jgm/pandoc/releases/download/{{ version }}/pandoc-{{ version }}-1-windows-x86_64.msi  # [win]
```

Comment:
Here is the problem: the filename is `pandoc-2.2.2-1-windows-x86_64.msi` (see https://ci.appveyor.com/project/condaforgePandoc/pandoc-feedstock/build/1.0.67#L450) and so in bld.bat the file will not be found because the filename there is referenced as `pandoc-%PKG_VERSION%-windows-x86_64.msi` -> I think the fix is making the version itself conditional on the OS

```
{% set version = "2.2.2" %} # [not win]
{% set version = "2.2.2-1" %} # [win]
```

(I have no idea if you can write the first line]